Podcast/DiscoverTopicViewController.swift

```diff
@@ -220,18 +226,20 @@ class DiscoverTopicViewController: DiscoverComponentViewController {
         relatedTopicsView.collectionView.reloadData()
     }
 
-    func fetchEpisodes(id: Int) {
+    func fetchEpisodes(id: Int, canPullToRefresh: Bool = false) {
 
         let topEpisodesForTopicEndpointRequest = DiscoverTopicEndpointRequest(requestType: .episodes, topicID: id, offset: offset, max: pageSize)
```

Comment:
let's reset the offset here instead of setup because this endpoint is what uses the offset